app/tests/integration/nimbus/utils/helpers.py

```diff
@@ -0,0 +1,86 @@
+import json
+import time
+
+import requests
+
+LOAD_DATA_RETRIES = 10
+LOAD_DATA_RETRY_DELAY = 1.0
+
+
+def load_graphql_data(query):
+    for retry in range(LOAD_DATA_RETRIES):
+        try:
+            return requests.post(
+                "https://nginx/api/v5/graphql",
+                json=query,
+                verify=False,
+            ).json()
+        except json.JSONDecodeError:
+            if retry + 1 >= LOAD_DATA_RETRIES:
+                raise
+            else:
+                time.sleep(LOAD_DATA_RETRY_DELAY)
+
+
+def load_targeting_configs():
+    data = load_graphql_data(
+        {
+            "operationName": "getConfig",
+            "variables": {},
+            "query": """
+                    query getConfig {
+                        nimbusConfig {
+                            targetingConfigs {
+                                label
+                                value
+                                applicationValues
+                            }
+                        }
+                    }
+                    """,
+        }
+    )
+    targeting_configs = []
+    for item in data["data"]["nimbusConfig"]["targetingConfigs"]:
+        if "DESKTOP" in item["applicationValues"]:
+            targeting_configs.append(item["value"])
+    return targeting_configs

```

Comment:
Is this being used anywhere?  This is filtering for desktop experiments but we want the opposite?

Reply:
I plan to replace the setup in `test_firefox_targeting.py` with this but that will follow this PR.

Reply:
Okay cool can you file the ticket and link it here as a followup pls? 🙏